typescript/cli/src/context/strategies/signer/MultiProtocolSignerManager.ts

```diff
@@ -50,11 +50,11 @@ export class MultiProtocolSignerManager {
     this.initializeStrategies();
   }
 
+  // TODO: readd Cosmos Native
```

Comment:
make codespell happy
```suggestion
  // TODO: re-add Cosmos Native
```